Assets/scss/components/quesionnaire.scss

```diff
@@ -0,0 +1,248 @@
+.cmp-questionnaire {
+  max-width: toRem(1175);
+  margin: auto;
+
+  @include max-screen(sm) {
+    max-width: toRem(767);
+  }
+
+  @include breakpoint-between(sm, md) {
+    max-width: toRem(1199);
+  }
+
+  &__container {
+    display: flex;
+    background-color: $grey94;
+
+    @include max-screen(sm) {
+      display: flex;
+      flex-direction: column;
+      overflow: auto;
+      background-color: $grey94;
+    }
+
+    @include breakpoint-between(sm, md) {
+      overflow: auto;
+      display: flex;
+      flex-direction: column;
+    }
+  }
+
+  &__image {
+    max-width: 50%;
+
+    @include max-screen(sm) {
+      max-width: 100%;
+      object-fit: cover;
+    }
+
+    @include breakpoint-between(sm, md) {
+      max-width: 100%;
+      object-fit: cover;
+    }
+  }
+
+  &__form {
+    margin-block-start: toRem(40);
+    margin-inline: toRem(40) toRem(50);
+
+    @include breakpoint-between(sm, md) {
+      margin-inline-end: toRem(100);
+    }
+  }
+
+  &__title {
+    text-align: start;
+    margin-block-end: toRem(30);
+
+    @include breakpoint-between(sm, md) {
+      text-align: center;
+    }
+  }
+
+  &__heading {
+    line-height: 1.32;
+    text-wrap: balance;
+    font-size: toRem(24);
+    font-weight: $font-bold;
+    font-style: normal;
+    color: $black;
```

Comment:
same

Reply:
Changed it to css variable.Thanks for the feedback.